geopandas/_compat.py

```diff
@@ -21,179 +17,13 @@
 
 
 # -----------------------------------------------------------------------------
-# Shapely / PyGEOS compat
+# Shapely / GEOS compat
 # -----------------------------------------------------------------------------
 
 
-SHAPELY_GE_182 = Version(shapely.__version__) >= Version("1.8.2")
-SHAPELY_GE_20 = Version(shapely.__version__) >= Version("2.0.0.dev0")
```

Comment:
Maybe could keep this single one. I assume we will have to add `SHAPELY_GE_21` soon, and then it's easier copy/paste

Reply:
For which reason? As a template?